src/unstract/sdk/file_storage/env_helper.py

```diff
@@ -0,0 +1,46 @@
+import json
+import logging
+import os
+
+from unstract.sdk.exceptions import FileStorageError
+from unstract.sdk.file_storage.constants import CredentialKeyword, StorageType
+from unstract.sdk.file_storage.impl import FileStorage
+from unstract.sdk.file_storage.permanent import PermanentFileStorage
+from unstract.sdk.file_storage.provider import FileStorageProvider
+from unstract.sdk.file_storage.shared_temporary import SharedTemporaryFileStorage
+
+logger = logging.getLogger(__name__)
+
+
+class EnvHelper:
+    @staticmethod
+    def get_storage(storage_type: StorageType, env_name: str) -> FileStorage:
```

Comment:
please add `docsring`

Reply:
Will take it up in the next PR @muhammad-ali-e Anyway I think I need to add them for helper.py also. I will do both together...